titan-server/src/bin/main.rs

```diff
@@ -1,16 +1,29 @@
-#![feature(proc_macro_hygiene, decl_macro)]
+#![feature(plugin, decl_macro, custom_attribute, proc_macro_hygiene)]
 
 extern crate libtitan;
 #[macro_use] extern crate rocket;
-extern crate rocket_contrib;
+use rocket::fairing::AdHoc;
+#[macro_use] extern crate rocket_contrib;
```

Comment:
One more comment: The `#[macro_use]` here is unnecessary since the binary doesn't use any of the macros supplied by the `rocket_contrib` crate.